beacon-chain/blockchain/chain_info.go

```diff
@@ -484,3 +482,66 @@ func (s *Service) Ancestor(ctx context.Context, root []byte, slot primitives.Slo
 func (s *Service) SetGenesisTime(t time.Time) {
 	s.genesisTime = t
 }
+
```

Comment:
I think all these methods specific to forkchoice should be in their own file ex: `forkchoice_chain_info` or something of that sort

Reply:
👍